testharness/src/test/java/bio/terra/landingzone/db/DeployedLandingZoneDaoTest.java

```diff
@@ -26,6 +26,7 @@ public class DeployedLandingZoneDaoTest extends LibraryTestBase {
   private static final String VERSION = "v1";
   private static final String DISPLAY_NAME = "lzDisplayName";
   private static final String DESCRIPTION = "lzDescription";
+  private static final String REGION = "eastus";
```

Comment:
Do any of these tests assert that the region is being stored and read back correctly? If not at least one should be made to do so.